### ISSUE TITLE
griefing: gestione problemi e soluzioni

### ISSUE THREAD

**Issue by pandaro** [MinetestItalia/minetest_italia#15]
Apro un issues per gestire i problemi derivanti da griefing:
1. problemi di secchi di lava e acqua:
ipotesi: aggiungere un privilegio di uso per i secchi, chi non è autorizzato non li può usare.
i privilegi te li guadagni con il tempo e l'affidabilità.
avanti con i suggerimenti

**Comment by H4mlet** [comment]
Concordo con l'idea del privilegio, il modulo tramite cui metterlo in pratica è [Bucket Privs](https://github.com/ChaosWormz/bucket_privs).

Per quanto riguarda il criterio di assegnazione del privilegio, meglio parlarne nel forum dello staff.

**Comment by H4mlet** [comment]
Ho messo alla prova Bucket Privs, funziona, ma blocca indiscriminatamente tutti i secchi.
Ho scoperto un altro modulo che permette una gestione piu' mirata, distinguendo tra secchi d'acqua e secchi di lava, e che oltre a impedire l'uso dei secchi rimuove i nodi di lava o acqua piazzati da chi non ha il permesso. Questo permette per esempio di consentire i secchi d'acqua e proibire quelli di lava, oltre a tenere il server "pulito in automatico".
L'ho messo alla prova, l'ho aggiornato per le nostre esigenze; apro un pull request in ramo di sviluppo.
